packages/mdc-select/index.js

```diff
@@ -40,8 +44,14 @@ class MDCSelect extends MDCComponent {
    */
   constructor(...args) {
     super(...args);
-    /** @private {?Element} */
+    /** @private {?HTMLElement} */
```

Comment:
Doesn't `HTMLElement` have a tendency to fail closure validation whereas `Element` works?